ports/esp32/README.md

```diff
@@ -195,7 +202,7 @@ quickly call `wlan_connect()` and it just works):
 ```python
 def wlan_connect(ssid='MYSSID', password='MYPASS'):
     import network
-    wlan = network.WLAN(network.WLAN.IF_STA)
+    wlan = network.WLAN(network.STA_IF)
```

Comment:
`network.WLAN.IF_STA` is actually preferred here, the top-level namespace constants like `network.STA_IF` are deprecated and will be removed at some point in the future. (This one was only changed recently in 285e1d0b)

Reply:
Corrected in newly commit